sdk/core/Azure.Core/src/Internal/SpanOfCharsExtensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Core
+{
+    internal static class SpanOfCharsExtensions
```

Comment:
Alternative to static helpers can be something like [`ValueStringBuilder`](https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/System/Text/ValueStringBuilder.cs), but I would prefer to make code that uses these methods look more low level to avoid possible misuse of array on stack

Reply:
`DefaultInterpolatedStringHandler` is good enough. Removing this custom helpers.